docs/ZKPool-2.0/technology/moludar-Galactic-Network-overview.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
This page http://localhost:3000/ZKPool-2.0/technology/moludar-Galactic-Network-overview
did not use the updated images I sent you yesterday?